qiita_db/sql_connection.py

```diff
@@ -150,16 +150,32 @@ def _open_connection(self):
 
         try:
             self._connection = connect(**args)
-        except Exception:
-            # catch any exception and raise as runtime error
-            # attempt to point the user to their config file
-            error_text = ("Can't connect to database; user `%s` is not known. "
-                          "Your QIITA configuration file, located at `%s` must"
-                          " specify the correct user, via the `USER` parameter"
-                          " under the `[postgres]` header. Please refer to "
-                          "`INSTALL.md` in the QIITA base directory.")
-            error_text = error_text % (qiita_config.user, qiita_config.conf_fp)
-            raise RuntimeError(error_text)
+        except Exception as e:
```

Comment:
I just read through the psycopg2 documentation and we should be able to look for specific exceptions here, see the exceptions here:

http://initd.org/psycopg/docs/module.html#psycopg2.InterfaceError 

I'm up to discuss this.